index.html

```diff
@@ -5321,6 +5322,15 @@ <h3>Protocol Bindings</h3>
             </span>
         </li>
     </ul>
+
+    <p class="note">
+        Optimally, the protocols used are listed as a scheme in the IANA registry [[?IANA-URI-SCHEMES]]). 
+        This guarantees a unique <a>Protocol Binding</a> assignment. In case the desired protocol is not yet 
+        registered with IANA, it is recommended to follow the schema rules of the protocol specifications, if available.
+        Otherwise, established implementations of the protocols specify a defacto scheme that should also be followed for 
+        TD definitions. If an IANA entry is not available, the <a>Protocol Binding</a> document will 
+        provide a recommended schema value to allow unique protocol identification in the context of WoT.
+    </p>
```

Comment:
I am not 100% sure why this note was added to this PR.

Having said that, I seldomly read "Optimally" ... more often "Ideally" .. but this should be judged by a native speaker. 

Moreover, I believe "defacto" should become "de facto" or "de-facto"

Reply:
strange, this is showing up here. This is related to PR #1309 

I will remove it here again